### ISSUE TITLE
Windows 10 Quarantined - Trojan

### ISSUE THREAD

**Issue by EgonTwo** [TheWaveWarden/odin2#370]
Threat detected: Trojan:Win32/Wacatac.D5!ml
Alert level: Severe
Date: 2020-12-16 19:24
Category: Trojan
Details: This program is dangerous and executes commands from an attacker.

Affected items: file: C:\Users\{my account}\Desktop\SW in\VSTi Odin2 (2.2.4).exe

**Comment by TheWaveWarden** [comment]
Hi @EgonTwo ,
Is this the built in windows defender, or a separate virus engine?

I write this as well in the FAQ on my website: The installer is not a virus. All current installers are built using teh Azure cloud with a fresh Windows image which is provided by Microsoft themselves. I see the chance of catching a virus there to be next to impossible. Yet still virus engines randomly flag the installer as a virus.

I see it is a problem still since you can not install.

**Comment by EgonTwo** [comment]
Hello WaveWarden,

It was the windows defender (OS built in) virus detector that raised this alert. And it didn't happen until a few weeks after I had downloaded the Odin installer. I can't find it on my computer anymore, presumably because it has been quarantined.

I just downloaded it again and scanned it twice with Windows Defender with no virus alert detected. I wonder if the other one I had downloaded had become infected on my computer (is that possible?) or it was just a freak error by windows defender?

Thanks, E
